src/utils/constants.hxx

```diff
@@ -94,9 +94,9 @@
 #else  // native C
 
 #define DECLARE_CONST(name)                                                    \
-    EXTERNC extern int _sym_##name;                                            \
+    EXTERNC extern char _sym_##name;                                           \
```

Comment:
Are there any concerns with limiting to 8bit values vs 32bit from int? I'm not certain it will make a difference since we use ptrdiff_t when retrieving the values.

Reply:
The value is the address of this symbol. That address is 32-bit in a 32-bit processor.

Reply:
Ok that is what I was thinking but wasn't 100% sure on